tests/lifecycle/tests/lifecycle_pod_recreation.go

```diff
@@ -0,0 +1,168 @@
+		err = globalhelper.ValidateIfReportsAreValid(
+			lifeparameters.PodRecreationDefaultName,
+			globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred())
+
+	})
+
+	// 47406
+	It("Multiple deployments with PodAntiAffinity, replicas < schedulable nodes", func() {
+		scheduleableNodes := nodes.GetNumOfReadyNodesInCluster(globalhelper.APIClient)
+
+		// at least one "clean of any reosource" worker is needed.
+		maxPodsPerDeployment := (scheduleableNodes / 2) - 1
+		By("Define & create first deployment")
+		firstDeploymentStruct := deployment.RedefineWithPodAntiAffinity(
+			lifehelper.DefineDeployment(maxPodsPerDeployment, 1, "lifecycleputone"),
+			lifeparameters.TestDeploymentLabels)
+
+		err := globalhelper.CreateAndWaitUntilDeploymentIsReady(firstDeploymentStruct, lifeparameters.WaitingTime)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Define & create second deployment")
+		secondDeploymentStruct := deployment.RedefineWithPodAntiAffinity(
+			lifehelper.DefineDeployment(maxPodsPerDeployment, 1, "lifecycleputtwo"),
+			lifeparameters.TestDeploymentLabels)
+
+		err = globalhelper.CreateAndWaitUntilDeploymentIsReady(secondDeploymentStruct, lifeparameters.WaitingTime)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start lifecycle lifecycle-pod-recreation test")
+		err = globalhelper.LaunchTests(
+			[]string{lifeparameters.LifecycleTestSuiteName},
+			lifeparameters.SkipAllButPodRecreationRegex)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			lifeparameters.PodRecreationDefaultName,
+			globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred())
+
+	})
+
+	// 47407
+	It("One deployment with PodAntiAffinity, replicas = schedulable nodes [negative]", func() {
+		By("Define & create deployment")
+		deploymentStruct := deployment.RedefineWithPodAntiAffinity(
+			lifehelper.DefineDeployment(
+				nodes.GetNumOfReadyNodesInCluster(globalhelper.APIClient),
+				1, "lifecycleput"), lifeparameters.TestDeploymentLabels)
+
+		err := globalhelper.CreateAndWaitUntilDeploymentIsReady(deploymentStruct, lifeparameters.WaitingTime)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start lifecycle lifecycle-pod-recreation test")
+		err = globalhelper.LaunchTests(
+			[]string{lifeparameters.LifecycleTestSuiteName},
+			lifeparameters.SkipAllButPodRecreationRegex)
+		Expect(err).To(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			lifeparameters.PodRecreationDefaultName,
+			globalparameters.TestCaseFailed)
+		Expect(err).ToNot(HaveOccurred())
+
+	})
+
+	// 47408
+	It("multiple deployments with PodAntiAffinity, replicas = schedulable nodes [negative]", func() {
+		scheduleableNodes := nodes.GetNumOfReadyNodesInCluster(globalhelper.APIClient)
+
+		// all nodes will be scheduled with a pod.
+		maxPodsPerDeployment := (scheduleableNodes / 2)
```

Comment:
You should consider an odd numer of schedulable deployments. In case schedulable nodes is 5, the TNF tc will pass, instead of fail as you're expecting in this QE TC.